parsers/parserUtil.ts

```diff
@@ -49,7 +47,7 @@ String.prototype.removeMetrics = function() {
 };
 
 String.prototype.removeAlergens = function() {
-    return this.replace(/\s*[\s\(\d,\)]+$/g, "");

```

Comment:
eslint was arguing that this escaping is not necessary... but I can double check

Reply:
true, it is not necessary, i removed my comment